src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewtakes/view/ViewTakesStyles.kt

```diff
@@ -37,13 +43,18 @@ class ViewTakesStyles : Stylesheet() {
         takeFlowPane {
             borderColor += box(Color.LIGHTGRAY)
             borderWidth += box(0.px, 0.px, 0.px, 0.px)
-            backgroundColor += Color.TRANSPARENT
+            backgroundColor += AppTheme.colors.defaultBackground
             spacing = 10.px
-            padding = box(20.px)
+            padding = box(10.px, 150.px)
             vgap = 16.px
             hgap = 16.px
         }
 
+        scrollpane {
+            maxHeight = Screen.getPrimary().visualBounds.height.px / 2.5
```

Comment:
This feels like an awkward way to do this; I would prefer if you want to set height to be tied to a percentage, use gridpane for the parent

Reply:
i'll use a percentage. I need the anchorpane as the parent so that i can do the drag and drop

Reply:
yeah but you need the percentage to be of the parent, not the monitor- it would make things buggy with multiple monitors, resizing, etc

Why does the drag and drop have to happen in an anchorpane?

Reply:
i think the anchorpane was necessary to show the take cards being dragged around.